tests/vmi_configuration_test.go

```diff
@@ -3391,3 +3391,28 @@ func createBlockDataVolume(virtClient kubecli.KubevirtClient) (*cdiv1.DataVolume
 
 	return virtClient.CdiClient().CdiV1beta1().DataVolumes(testsuite.GetTestNamespace(nil)).Create(context.Background(), dataVolume, metav1.CreateOptions{})
 }
+
+func getProcessName(pod *k8sv1.Pod, pid string) (string, error) {
+	if pid == "" {
+		return "", fmt.Errorf("PID cannot be empty")
+	}
+
+	virtClient, err := kubecli.GetKubevirtClient()
+	if err != nil {
+		return "", fmt.Errorf("failed to get KubeVirt client: %w", err)
+	}
+
+	fPath := "/proc/" + pid + "/comm"
+	cmd := []string{"cat", fPath}
+	output, err := exec.ExecuteCommandOnPod(
+		virtClient,
+		pod,
+		"compute",
+		cmd,
+	)
+	if err != nil {
```

Comment:
I'm not sure I see the benefit of the second commit.

Reply:
Add more error handling like we talked about, I can remove if you feel it's not needed

Reply:
I'm not sure I understand to what you are referring. We mentioned that helper functions are better when they do not use `Expect`, so that they can be used outside Ginkgo, too.

Reply:
but you said in general that a good func/test is one that throw accurate error. maybe I'm wrong.
I will remove the commit

Reply:
removed the second commit

Reply:
> but you said in general that a good func/test is one that throw accurate error

I don't recall the context. Typically I vote against adding validations. In this case, since `pid` is blindly passed to a helper function, this function should not bother to validate it.